jooby/src/main/java/org/jooby/handlers/AssetHandler.java

```diff
@@ -185,11 +187,20 @@ public AssetHandler cdn(final String cdn) {
   }
 
   /**
-   * @param maxAge Set the cache header max-age value in seconds.
+   * @param maxAge Set the cache header max-age value.
    * @return This handler.
    */
-  public AssetHandler maxAge(final long maxAge) {
-    this.maxAge = maxAge;
+  public AssetHandler maxAge(final Optional<Duration> maxAgeOpt) {
```

Comment:
> If we allow a user to set this... why allow an possible empty object? We shouldn't.

The initial value being to be empty, setting a duration activate the feature. The purpose of this method is to allow the user to either activate or deactivate the feature.

> Also and to keep backward compatibility.. we should keep the long version as it was.

Ok, no problem with that.